tests/test_game.py

```diff
@@ -192,6 +194,41 @@ def test_play_year_of_plenty_no_year_of_plenty_card():
         game.execute(action_to_execute)
 
 
+def test_play_monopoly_no_monopoly_card():
+    players = [SimplePlayer(Color.RED), SimplePlayer(Color.BLUE)]
+    game = Game(players)
+
+    action_to_execute = Action(players[0], ActionType.PLAY_MONOPOLY, Resource.ORE)
+
+    with pytest.raises(ValueError):  # no monopoly
+        game.execute(action_to_execute)
+
+
+def test_play_monopoly_player_steals_cards():
```

Comment:
For this test, I wanted to show that `player_to_act` stole the ORE and left the other resources intact (in this case just WHEAT). I was too lazy to add some other resources in there, but can make this test stronger if necessary